docs/ibc/overview.md

```diff
@@ -10,131 +10,147 @@ Learn what IBC is, its components and use cases. {synopsis}
 
 ## What is the Interblockchain Communication Protocol (IBC)?
 
-This document serves as a guide for developers who want to write their own Inter-blockchain
-Communication Protocol (IBC) applications for custom use cases.
+This document serves as a guide for developers who want to write their own Inter-Blockchain
+Communication protocol (IBC) applications for custom use cases.
 
 Due to the modular design of the IBC protocol, IBC
-application developers do not need to concern themselves with the low-level details of clients,
-connections, and proof verification. Nevertheless a brief explanation of the lower levels of the
-stack is given so that application developers may have a high-level understanding of the IBC
-protocol. Then the document goes into detail on the abstraction layer most relevant for application
-developers (channels and ports), and describes how to define your own custom packets, and
+application developers do not need to be concerned with the low-level details of clients,
+connections, and proof verification. 
+
+This brief explanation of the lower levels of the
+stack is provided to give application developers a high-level understanding of the IBC
+protocol. Details on the abstraction layer are most relevant for application
+developers (channels and ports) and describe how to define custom packets and
 `IBCModule` callbacks.
 
-To have your module interact over IBC you must: bind to a port(s), define your own packet data (and
-optionally acknowledgement) structs as well as how to encode/decode them, and implement the
-`IBCModule` interface. Below is a more detailed explanation of how to write an IBC application
+To have your module interact over IBC you must: 
+
+- Bind to a port or ports
+- Define your own packet data
+- Define optional acknowledgement structs
+- Know how to encode and decode the packet data
+- Implement the `IBCModule` interface
```

Comment:
@crodriguezvega in this case, we can apply the Google developer documentation technical content style standards and add a period (.) at the end of each list item since the items include a verb.

My preference is to omit punctuation at the end of sentence fragments in lists; the key point is to be consistent throughout the content set. 

https://developers.google.com/style/periods?hl=en#bulleted-lists